crafters/image/AlbumentationsCrafter/README.md

```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
 
 > Note that Albumentation's transform are meant to be random, which is usually not what you want in indexing/search context. The crafter automatically adds `always_apply=True` to all transforms, but you should watch out for other sources of "randomness" (see AlbumentationsCrafter documentation for more detail)
 
+## Usage
+
+Users can use Pod images in several ways:
+
+1. Run with Docker (`docker run`)
+   ```bash
+    docker run jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in 55555 --port-out 55556
+    ```
+
+2. Run with Flow API
+   ```python
+    from jina.flow import Flow
+    f = (Flow()
+        .add(name='my_crafter', uses='docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7', port_in=55555, port_out=55556))
+    ```
+
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-out 55556
+    ```
+
+4. Conventional local usage with `uses` argument
+    ```bash
+    jina pod --uses hub/example/imagecrafter.yml --port-in 55555 --port-out 55556
```

Comment:
It's not clear what is `hub/example/imagecrafter.yml`.

Reply:
this is the config yml file for image crafter, renamed it